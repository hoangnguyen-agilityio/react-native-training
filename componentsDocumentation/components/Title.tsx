import React from 'react';
import {StyleSheet, Text} from 'react-native';

const Title = (props: any) => <Text style={styles.title} {...props} />;

const styles = StyleSheet.create({
  title: {
    fontSize: 18,
    marginBottom: 12,
  },
});

export default Title;
