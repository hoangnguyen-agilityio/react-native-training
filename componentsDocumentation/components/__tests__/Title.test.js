import React from 'react';
import renderer from 'react-test-renderer';
import Title from '../Title';

test('renders correctly', () => {
  const tree = renderer.create(<Title />).toJSON();
  expect(tree).toMatchSnapshot();
});