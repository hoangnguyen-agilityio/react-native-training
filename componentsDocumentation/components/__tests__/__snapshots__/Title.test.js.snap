// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Object {
      "fontSize": 18,
      "marginBottom": 12,
    }
  }
/>
`;
